src/test/java/testsupport/matchers/ZeroInvocationsMatcher.java

```diff
@@ -26,7 +27,7 @@ public boolean matches(Object item) {
     @Override
     public void describeMismatch(Object item, final Description description) {
         description.appendText("had these: ");
-        for (Invocation invocation : new MockUtil().getMockHandler(item).getInvocationContainer().getInvocations())
+        for (Invocation invocation : ((InvocationContainerImpl) MockUtil.getMockHandler(item).getInvocationContainer()).getInvocations())
```

Comment:
Hacky, no idea if there is a better way.

Reply:
I was able to find `MockUtil#getInvocationHandler`, which does the trick!